### PR TITLE
Update metadata too

### PR DESCRIPTION
**Pull request by ironsidevsquincy**

### CODE REVIEW

File: cli.js [hunk 1]
```diff
@@ -22,17 +34,37 @@ function republishImageSet(url) {
...
-		.then(function(image) {
-			return fetch("http://binary-ingester-iw-uk-p.svc.ft.com/ingest", {
-				method: 'POST',
-				timeout: 3000,
-				body: '{ "contentUri" : "http://methode-image-model-transformer-iw-uk-p.svc.ft.com/image/model/' + image.contentOrigin.originatingIdentifier + '" }',
-				headers: {
-					'Content-Type': 'application/json'
-				}
-			});
+            var imageUpdates = imageSet.members.map(function (image) {
```
Comment (matthew-andrews) [inline cli.js]: being picky but i think `var` can be `const`

File: cli.js [hunk 2]
```diff
@@ -22,17 +34,37 @@ function republishImageSet(url) {
...
+                var imageId = image.id.replace('http://api.ft.com/content/', '');
```
Comment (matthew-andrews) [inline cli.js]: again, `var` => `const`

File: cli.js [hunk 3]
```diff
@@ -22,17 +34,37 @@ function republishImageSet(url) {
...
+	                    .then(function(response) {
+	                        if (!response.ok) {
+	                    		console.log('ERROR')
+	                            throw new Error("Unable to refeed image '" + imageId + "'");
+	                        }
+	                        console.log("Successfully refed image '" + imageId + "'");
+	                    })
+                ];
+            });
+            var imageSetId = imageSet.id.replace('http://www.ft.com/thing/', '');
```
Comment (matthew-andrews) [inline cli.js]: `var` => `const`

File: cli.js [hunk 4]
```diff
@@ -43,28 +75,22 @@ program
 	.action(function(uuid) {
 		fetchCapiV2('http://api.ft.com/content/' + uuid)
 			.then(function(article) {
-				let promises = article.bodyXML.match(/ImageSet" url="http:\/\/api\.ft\.com\/content\/([a-z-0-9]{3,})"/g)
+				var imageIds = article.bodyXML.match(/ImageSet" url="http:\/\/api\.ft\.com\/content\/([a-z-0-9]{3,})"/g)
```
Comment (matthew-andrews) [inline cli.js]: can be `const`

File: cli.js (hunk 1 above)
Comment (ironsidevsquincy) [inline cli.js]: yep, need to get into the `const` world - will update them

**Pull request merged**